test/tests/shouldAdd.test.ts

```diff
@@ -28,13 +29,13 @@ it('should add a language and job using instance methods', async () => {
     languages: ['english'],
     uniqueId: 'unique-id'
   });
-  await user.addJob({ position: 'Dark Wizzard', title: 'Archmage' });
+  await user.addJob(new Job({ position: 'Dark Wizzard', title: 'Archmage' }));
```

Comment:
why did this get transformed into `new Job`?

Reply:
I just refactored it to get rid of the workaround that was present in `job.ts` where `titleInUppercase` was defined as an optional function, which was just an overall bad practice. It only affects tests and makes things cleaner.